src/plugins/vis_type_timeseries/public/application/visualizations/views/timeseries/utils/theme.test.ts

```diff
@@ -17,28 +17,30 @@
  * under the License.
  */
 
-import { getTheme } from './theme';
+import { getBaseTheme } from './theme';
```

Comment:
Are you trying to completely replace `getTheme` with `getBaseTheme`?

Reply:
What do you mean by replace it?

This was just a utility function to get the theme which was actually generating a base theme so I just updated the naming. 

Ideally, there is a `baseTheme` from `@elastic/charts`.  Then this function returns a `Partial<Theme>` using EUI that passes the value to the `Chart.theme` prop.